static/sass/_patterns_docs.scss

```
@@ -24,42 +24,6 @@
     }
   }
 
-  .l-docs-content {
-    flex: 5;
-    margin-top: 0;
-    overflow: hidden;
-
-    @media only screen and (max-width: $breakpoint-navigation-threshold) {
-      flex: auto;
-    }
-
-    .meta {
-      display: none;
-    }
-
-    details + p,
-    details + h3 {
-      margin-top: $sp-large;
-    }
-  }
-
-  .l-docs-row {
-    @extend .row;
-    margin-left: inherit;
-    padding: 0 $sp-xxx-large;
-
-    @media only screen and (max-width: $breakpoint-navigation-threshold) {
-      padding-left: $sph-outer--large;
-    }
-  }
-
-  .p-sidenav__toggle {
-    background-size: $sp-medium;
-    cursor: pointer;
-    float: right;
-    padding: $sp-x-large;
-  }
-
   .p-sidenav__body {
     padding: $sp-large / 2 $sp-large;
```

This raises a warning about the [slash divider operator being deprecated](https://sass-lang.com/documentation/breaking-changes/slash-div), and removed in the next major version - could use the new syntax here?

```suggestion
    padding: math.div($sp-large, 2) $sp-large;
```

You'd also need to add `@use "sass:math";` at the top of the file.